site/en/gemini-api/docs/vision.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<blockquote>Individual frames are 258 tokens, and audio is 32 tokens per second. With metadata, each second of video becomes ~300 tokens, which means a 1M context window can fit about 55.5 minutes of video.&nbsp;</blockquote><p> </p><p>Precision nit: When you approximate <code>~300 tokens</code>, you're using 1  degree of precision, then you switch to 3 degrees of precision for <code>55.5 minutes</code>, which is  quite specific, and slightly off the full calculation.</p><p> </p><p>Either stick with the single-digit approximation all the way through (e.g. "~300 tokens, which fits just under an hour of video") or be precise the whole way through (e.g. <code>290 tokens, giving 57mins of video</code>).</p><p> </p><p>Personally I prefer the approximate version, as users likely still need to fit a prompt, instructions, and possibly other media in.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/453/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
Done.